forui/lib/src/widgets/calendar/shared/entry.dart

```diff
@@ -5,60 +5,58 @@ import 'package:intl/intl.dart';
 import 'package:meta/meta.dart';
 import 'package:sugar/sugar.dart';
 
+import 'package:forui/forui.dart';
 import 'package:forui/src/foundation/tappable.dart';
-import 'package:forui/src/widgets/calendar/day/day_picker.dart';
-import 'package:forui/src/widgets/calendar/year_month_picker.dart';
 
 final _yMMMMd = DateFormat.yMMMMd();
 
 @internal
 abstract class Entry extends StatelessWidget {
   final FCalendarEntryStyle style;
-  final ValueWidgetBuilder<bool> builder;
+  final ValueWidgetBuilder<FTappableState> builder;
 
   factory Entry.day({
     required FCalendarDayPickerStyle style,
     required LocalDate date,
     required FocusNode focusNode,
     required bool current,
     required bool today,
-    required Predicate<LocalDate> enabled,
+    required Predicate<LocalDate> selectable,
     required Predicate<LocalDate> selected,
     required ValueChanged<LocalDate> onPress,
     required ValueChanged<LocalDate> onLongPress,
   }) {
-    final enable = enabled(date);
-    final select = selected(date);
+    final canSelect = selectable(date);
+    final isSelected = selected(date);
```

Comment:
Do you think it's better to just name this `selectable` and `selected`?

Reply:
I can't since they'll shadow the functions